packages/terra-form-input/src/Input.jsx

```diff
@@ -19,6 +19,10 @@ const propTypes = {
    * Whether the input is disabled.
    */
   disabled: PropTypes.bool,
+  /**
+   * Whether the input is incomplete. Input must also be required.
```

Comment:
Let's update our descriptions for isIncomplete to say:
```suggestion
   * Whether the input displays as Incomplete. Use when no value has been provided. _(usage note: `required` must also be set show display)_

```


\
Also update isInvalid to match and say:
```
   * Whether the input displays as Invalid. Use when value does not meet validation pattern.
```

Reply:
What does "`required` must also be set show display" mean?

Reply:
Also, in addition to the usage note in the documentation, is it a lot of effort for us to validate this prop against the fact that if this is set to true, then `required` _needs_ to be strictly true as well?

Reply:
Updated the `isInvalid` and `isComplete` prop descriptions for all: https://github.com/cerner/terra-core/pull/2670/commits/f3184124d28bce916fe72b76f0f19fa0aea8c0fd